src/lib/footer/Footer.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+	import { twMerge } from 'tailwind-merge';
+
+	let className = '';
+	export { className as class };
+	export let startClass = '';
+	export let middleClass = '';
+	export let endClass = '';
+	export let bottomClass = '';
+</script>
+
+<div
+	class={twMerge(
+		'max-w-screen-2xl bg-white p-4 text-sm dark:bg-gray-700 sm:p-10 2xl:mx-auto',
+		className
+	)}
+	{...$$restProps}
+>
+	<div class={'justify-between lg:flex'}>
+		<div class={('flex w-full md:w-auto', startClass)}>
```

Comment:
This still needs twMerge, I didn't mean to remove it from everything, just the two instances where it wasn't actually doing anything.